contao/templates/rsce/rsce_hyperlink_list_config.php

```diff
@@ -71,7 +71,33 @@
 					),
 					'inputType' => 'text',
 					'eval'      => array('maxlength'=>64, 'tl_class'=>'w50'),
-				)
+				),
+                'expert' => array(
```

Comment:
Maybe we should not block the field name `expert` and use the suffix `_legend` for groups.